docs/source/getting_started/install.rst

```diff
@@ -20,9 +20,11 @@ Prerequisites
 -------------
 
 You will need a working Python installation. FiftyOne currently requires
-**Python 3.7 - 3.10**.
+**Python 3.7 - 3.12** (3D features may not be available in 3.12 until
```

Comment:
IMO we can just remove the `open3d` note here. It's not even installed by default and if you don't have it you'll get a nice message saying "this feature required open3d to be installed". Then if you try to follow orders on Python 3.12, you'll figure out that it is either available or not.

Plus, are we really going to remember to update this when 3D is supported? 🤣

Reply:
I subscribed to that issue so I would get notifications but yeah you're right, was just trying to save someone a few minutes if they ran into this